java/HealthVault Mobile/HVSample/build.gradle

```diff
@@ -20,6 +20,8 @@ android {
 
 dependencies {
     compile(name:'android-sdk-simplexml-release', ext:'aar')
+    compile(name:'achartengine-1.1.0', ext:'jar')
```

Comment:
I see usage of the graph library but not of the achartengine library in this PR. Do we need/use a chart engine? If we don't we should remove this dependency and also the .jar in the libs director.